chain/chain/src/store.rs

```diff
@@ -33,6 +35,8 @@ const TAIL_KEY: &[u8; 4] = b"TAIL";
 const SYNC_HEAD_KEY: &[u8; 9] = b"SYNC_HEAD";
 const HEADER_HEAD_KEY: &[u8; 11] = b"HEADER_HEAD";
 const LATEST_KNOWN_KEY: &[u8; 12] = b"LATEST_KNOWN";
+const LARGEST_APPROVED_WEIGHT_KEY: &[u8; 23] = b"LARGEST_APPROVED_WEIGHT";
```

Comment:
```suggestion
const LARGEST_APPROVED_WEIGHT_KEY: &[u8] = b"LARGEST_APPROVED_WEIGHT";
```

Do you really need to pass define the length?